handsontable/src/core.js

```diff
@@ -3350,11 +3350,13 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
    * @function getCellMeta
    * @param {number} row Visual row index.
    * @param {number} column Visual column index.
+   * @param {object} options Execution options for the `getCellMeta` method.
+   * @param {boolean} [options.executeCellsFunction] If `false`, disables the execution of the `cells` function. Default: `true`.
    * @returns {object} The cell properties object.
    * @fires Hooks#beforeGetCellMeta
    * @fires Hooks#afterGetCellMeta
    */
-  this.getCellMeta = function(row, column) {
+  this.getCellMeta = function(row, column, options = { executeCellsFunction: true }) {
```

Comment:
I'd propose a different name for the option. For me, it's not about preventing the call of a `cells` function but rather preventing the execution of the [_DynamiCellMeta_](https://github.com/handsontable/handsontable/blob/develop/handsontable/src/dataMap/metaManager/mods/dynamicCellMeta.js) mod module.
```suggestion
  this.getCellMeta = function(row, column, options = { skipMetaExtension: false }) {
```

Reply:
Ok, I changed it